CMakeLists.txt

```diff
@@ -230,6 +230,15 @@ if(ENABLE_ASAN OR ENABLE_TSAN OR ENABLE_UBSAN)
   set(ENABLE_STANDALONE_CALCITE ON)
 endif()
 
+# Embedded database
+option(ENABLE_DBE "Enable embedded database" OFF)
+if(ENABLE_DBE)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
+  set(ENABLE_STANDALONE_CALCITE ON)
```

Comment:
Looks like this flag isn't connected to anything at the moment (looks like it wasn't removed from cmake/docs when the code using it was removed).

Reply:
I use this flag in the cmake command to build shared library for embedded database. It also defines the ENABLE_EMBEDDED_DATABASE macros which used in the funcannotations.h to set visibility.
In this case all static libraries should be compiled with -fPIC.

Reply:
It's just initial changes and tests will be added in the next PR with python code.